aws-datastore/src/main/java/com/amplifyframework/datastore/DataStoreConflictHandler.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.datastore;
+
+import androidx.annotation.NonNull;
+
+import com.amplifyframework.core.Consumer;
+import com.amplifyframework.core.model.Model;
+
+/**
+ * Handles conflicts between models observed during model synchronization in the DataStore.
```

Comment:
Hmm this is the third time i've seen this.. is it really "the Datastore" ?  Maybe it's intentional?

Reply:
Months back, we had decided on the stylization _DataStore_ as a product decision.

So, we laid to reset the Pascal-casing identifier for the product. However, I do not think we have standardized on the camel-cased and sreaming-snake sytlizations:

0. `dataStore`, or `datastore`?
1. `DATA_STORE` or `DATASTORE`?

IIRC, Android is ~uniformly using these stylizations, right now:
- `DataStore` (pascal)
- `dataStore` (camel), and
- `DATASTORE` (screaming snake.)